### ISSUE TITLE
Switching chart provider

### ISSUE THREAD

**Issue by gitbute** [khuedoan/homelab#94]
Hi there

As the k8s-at-home charts are marked as deprecated / archived and wont get updated regularly,
i propose we switch to something that still is maintained for the default app-stack.

I mainly played around with a combination of [https://github.com/truecharts](https://github.com/truecharts) and linuxserver.io images where i didnt found a suitable image. I mostly had good results and more up to date apps.

If you would like me to get some of the apps ported over, i will gladly make some pull requests.

Best regards
gitbute

**Comment by gitbute** [comment]
So, Truecharts decided to not support native helm deployments anymore, only on their Truenas SCALE solution.
It looks like we to search another common chart provider for these apps.

**Comment by khuedoan** [comment]
Worst case scenario we can maintain our own charts, as I did with the `cloudflared` chart (though it's a little out of date 😅).

**Comment by dotdiego** [comment]
k8s-at-home can be replaced with a library chart by one of k8s-at-home charts maintainer.
See more here : https://bjw-s.github.io/helm-charts/docs/app-template/introduction.html

I could not make it work with ArgoCD + Helm though. I used Kustomized Helm to make it work.

See below example for Jellyfin.

```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
helmCharts:
  - name: app-template
    repo: https://bjw-s.github.io/helm-charts/
    version: 1.3.0
    releaseName: &releaseName jellyfin
    namespace: &releaseNamespace jellyfin
    valuesFile: values.yaml
commonAnnotations:
  meta.helm.sh/release-name: *releaseName
  meta.helm.sh/release-namespace: *releaseNamespace
commonLabels:
  app.kubernetes.io/managed-by: Helm
```

**Comment by onedr0p** [comment]
I am a little biased here but I also think app-template is the way to go for the major of apps in the "selfhosted" catalog like jellyfin. All the "selfhosted" applications I have deployed in my GitOps repo use it https://github.com/onedr0p/home-ops/tree/main/kubernetes/apps/default

**Comment by khuedoan** [comment]
I had intended to switch to app-template since k8s-at-home was archived, but I was side tracked to other things :laughing: I'll go back to this project in the next few days

**Comment by khuedoan** [comment]
It's done, thanks everyone :tada:

- Jellyfin: switched to app-template, added some more \*arr services for _totally legal_ media management (5e58a21aecf9865e2f9fa7ff831a59cd5e698b88, 6d7255cca32f3b3933ad3e68999b8e40d76136d0, 1e4eae4894fa7620a3081ad6f644401731493ba3 and more)
- Paperless: switched to app-template + bug fixes (11e407e880762f105f30a493680b5763736f3186)
- Seafile: removed (0eb03e4)
